### DISCUSSION TITLE
How can I create an Isosurface map from a traffic simulation?

### DISCUSSION THREAD

**Discussion by oibeal** [Universite-Gustave-Eiffel/NoiseModelling#569] [Q&A] [answered]
Hello everyone!

I have been using noise modeling for a research problem related to the calculation of noise caused by city traffic. I have followed several of the tutorials and tried things on my own to generate different point and polygon maps based on different maps and simulations. I'm using version 4.0.4 of the program for it.

The problem I have encountered is the following:

I have a traffic simulation and an osm map. Which I load and from them I generate the noise map (a process similar to the one in the MATSim - GUI tutorial, only that I use a Delaunay Grid to generate the receivers instead of the activities). With this I get to the point where I have my noise map from the attenuation matrix (RESULT_GEOM as in the tutorial).

From here what I want is to be able to generate the CONTOURING_NOISE_MAP. Taking as input the values of the noise simulation. The problem is that my noise map is not valid as input for Create_Isosurface since it shows the values by time bins and what I need is a table of the LDEN type (which right now contains the attenuation matrix and not the noise levels) . So I suppose that some step has to be done differently to be able to generate a valid LDEN table or, failing that, perform some process with RESULT_GEOM as input to generate it.

What do you think about it, do you know how to solve this problem?

I thank you in advance for your replies and have a nice day !

**Comment by nicolas-f** [comment, marked as answer]
Hi,

You can create a isosurface for each time step or do a weighted energetic sum of each time step to produce a LDEN column.

![image](https://user-images.githubusercontent.com/1382241/231393485-91a8cb6d-349c-405f-bfa0-4b9c69595a7b.png)
(source: https://www.bruitparif.fr/les-indicateurs-de-bruit1/)

**Reply by oibeal** [reply]
Thank you very much for the reply.

Yes. Something like that is what I had seen on the internet that could be done in these cases. I understand the point of applying the formula to obtain the LDEN of each receiver with the result data. This gives me the following question: Is there a way to do it in NoiseModelling currently or is it necessary to create my own?

If the answer is that it is necessary to create it. I understand that what is needed to do is create a valid table (at least the columns: IDRECEIVER🔑, THE_GEOM and LAEQ) for the _Create_Isosurface_ block where the LAEQ column is the result of applying the LDEN formula on the RESULTS_GEOM table to generate a valid CONTOURING_NOISE_MAP table with the desired values, am I right?

**Reply by nicolas-f** [reply]
> If the answer is that it is necessary to create it. I understand that what is needed to do is create a valid table (at least the columns: IDRECEIVERkey, THE_GEOM and LAEQ) for the _Create_Isosurface_ block where the LAEQ column is the result of applying the LDEN formula on the RESULTS_GEOM table to generate a valid CONTOURING_NOISE_MAP table with the desired values, am I right?

There is no NoiseModelling wps to generate lden from timestamp table.

Yes you have to generate the columns expected by create isosurface wps. You can do that using SQL or your favourite programming language.

https://noisemodelling.readthedocs.io/en/latest/NoiseModelling_db.html#connect-dbeaver-to-your-database

**Reply by oibeal** [reply]
I get it, great! Thank you a lot.
